include/grpc/support/dynamic_annotations.h

```diff
@@ -0,0 +1,67 @@
+/*
```

Comment:
Since this file is for internal use only, it probably doesn't need to be in the include tree.  Instead, I suggest moving this file to src/core/lib/iomgr.